vendor/squizlabs/php_codesniffer/CodeSniffer.conf

```diff
@@ -1,6 +0,0 @@
-<?php
- $phpCodeSnifferConfig = array (
-  'installed_paths' => '../../../vendor/wp-coding-standards/wpcs,../../../vendor/wimg/php-compatibility,../../../vendor/eventespresso/ee-coding-standards',
-  'default_standard' => 'EventEspresso',
-)
-?>
```

Comment:
It's really puzzling how a `vendor` file is getting committed here?  Do you know how this happened?

Reply:
> It's really puzzling how a vendor file is getting committed here? Do you know how this happened?

I'm still curious about this.  It's a bit alarming that a vendor path change was committed when it _should_ be ignored.

Reply:
phpStorm keeps telling me that file is under version control and I keep adding it to my ignore list. I have told phpStorm to ignore it, I have ignored it via command line, and I have gone into my ignore file manually and added. I have also done the same with the entire `/vendor/` folder.

But every so often, usually after switching branches, my IDE tells me once again that that file is under VC and asks if I once again want to ignore it, which I always do. So something funky is likely stuck in my phpStorm config, or something else. Let's not worry about it.